sdk/pyproject.toml

```diff
@@ -39,6 +39,8 @@ watchdog = "^3.0.0"
 python-docx = "^1.1.0"
 python-jose = "^3.3.0"
 authlib = "^1.3.0"
+aiostream = "^0.5.2"
+sse-starlette = "^1.8.2"
```

Comment:
Do we need sse-starlette? fastapi has EventSourceResponse in it builtin from the response side